pkg/controller/clusterstate/api/node_info.go

```diff
@@ -133,6 +138,7 @@ func (ni *NodeInfo) SetNode(node *v1.Node) {
 	ni.Labels = NewStringsMap(node.Labels)
 	ni.Unschedulable = node.Spec.Unschedulable
 	ni.Taints = NewTaints(node.Spec.Taints)
+	ni.IsReady = node.Status.Conditions[0].Status
```

Comment:
I think the function`GetCondition()` is needed here?

Reply:
Done, thanks